docs/access/stratum1.md

```diff
@@ -0,0 +1,357 @@
+server that was specified when creating the repository replica, the initial synchronisation may take a while.
+
+The time required for the initial synchronisation is heavily dependent on the size of the repository,
+and the available network bandwidth to the synchronisation server.
+
+### Periodic synchronisation
+
+To ensure that updates to the contents of the CernVM-FS repository are synchronised
+automatically to the Stratum-1 replica server, we should set up a cron job to do periodic synchronisation by running `cvmfs_server snapshot -a`.
+
+#### Log rotation
+
+Before setting up a cron job, we first need to configure log rotation, or running `snapshot -a` will fail with:
+```
+/etc/logrotate.d/cvmfs does not exist!
+```
+
+Create `/etc/logrotate.d/cvmfs` with the following contents:
+
+``` { .bash .copy }
+/var/log/cvmfs/*.log {
+    weekly
+    missingok
+    notifempty
+}
+```
+
+#### Cron job
+
+To synchronize all active replica repositories every 5 minutes, we can create a cron job `/etc/cron.d/cvmfs_stratum1_snapshot` that runs `cvmfs_server snapshot -a -i`:
+
+``` { .bash .copy }
+*/5 * * * * root output=$(/usr/bin/cvmfs_server snapshot -a -i 2>&1) || echo "$output"
+```
+
+The `-a` option enables synchronisation of *all active* replica repositories,
+while `-i` indicates that that repositories for which an initial snapshot has not been run should be skipped.
+
+To verify that periodic synchronisation is working correctly,
+check the contents of the log file:
+
+``` { .bash .copy }
+/var/log/cvmfs/snapshots.log
+```
+
+### More information
+
+For more information on the setup and configuration of a Stratum 1 replica server,
+see the CernVM-FS documentation, in particular the following sections:
+
+- [Notable CernVM-FS Server Locations and Files](https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#sct-serveranatomy)
+- [CernVM-FS Server Infrastructure](https://cvmfs.readthedocs.io/en/stable/apx-serverinfra.html)
+
+
+### Client system configuration
+
+To actually use the "private" Stratum 1 replica server that has been set up
+we need to change the configuration on each CernVM-FS client system.
+
+#### Stratum 1 replica server(s)
+
+The `CVMFS_SERVER_URL` configuration setting on a client system:
+
+- Is a string value with a semicolon-separated (`;`) list of known Stratum 1 servers;
+- Should be enclosed in quotes;
+- Specifies each Stratum 1 as a URL that starts with `http://`, and ends with `/cvmfs/@fqrn@`
+
+For example:
+
+```
+CVMFS_SERVER_URL="http://s1.test.eu/cvmfs/@fqrn@;http://s1.test.us/cvmfs/@fqrn@"
+```
+
+The `@fqrn@` substring is replaced by CernVM-FS with the *fully qualified repository name*, like `software.eessi.io`.
+
+`CVMFS_SERVER_URL` should be specified in the *domain-specific configuration file* in `/etc/cvmfs` that is
+relevant for the CernVM-FS repository we have replicated on our Stratum 1.
+
+For `software.eessi.io`, we should add the following to `/etc/cvmfs/domain.d/eessi.io.conf`:
```

Comment:
Though this is not incorrect, I find it cleaner to not touch the original config file, but instead you can make a local config file (this will be automatically picked up by CVMFS, and with lower priority, see `/etc/cvmfs/default.conf`) `etc/cvmfs/domain.d/<your_domain>.local` or `/etc/cvmfs/config.d/<your_repository>.local`. You can then also use a nice trick by doing:
```
CVMFS_SERVER_URL="http://privatestratum1/cvmfs/@fqrn@;$CVMFS_SERVER_URL"
```

Reply:
fixed in 7c9c694, not mentioning `$CVMFS_SERVER_URL` trick for now, can do in follow-up PR